certified-connectors/ProcessStreet/apiDefinition.swagger.json

```diff
@@ -0,0 +1,931 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Process Street",
+    "description": "Process Street is a modern process management platform for building, sharing, and automating your core processes. You can perform various actions such as starting, finding, and updating a workflow run in Process Street. Or trigger actions when a workflow starts or task status changes.",
+    "version": "1.0",
+    "contact": {
+      "name": "Process Street Support",
+      "url": "https://www.process.st/help",
+      "email": "support@process.st"
+    }
+  },
+  "host": "public-api.process.st",
+  "basePath": "/api/v1.1",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://www.process.st"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://www.process.st/privacy"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Business Management;Productivity"
+    }
+  ],
+  "paths": {
+    "/workflow-runs": {
+      "get": {
+        "summary": "Find workflow runs",
+        "x-ms-summary": "Find workflow runs",
+        "description": "Returns the latest 200 active workflow runs of an organization, showing the most recently updated first.",
+        "operationId": "listWorkflowRuns",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "workflowId",
+            "description": "The workflow Id.",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "x-ms-summary" : "Workflow Id",
+            "x-ms-url-encoding" : "single",
+            "x-ms-dynamic-values" : {
+              "operationId" : "listWorkflows",
+              "value-collection" : "workflows",
+              "value-path" : "id",
+              "value-title" : "name"
+            }
+          },
+          {
+            "name": "name",
+            "x-ms-summary": "Name",
+            "description": "The workflow name to search for (Optional).",
+            "in": "query",
+            "required": false,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/ListWorkflowRunsResponse"
+            }
+          },
+          "400": {
+            "description": "Invalid value for: query parameter workflowId, Invalid value for: query parameter name"
```

Comment:
We had them originally but I got rid of them. That's because `paconn` raises a warning if you have an operation with more than one response. E.g:

```Warning : paths/~1workflow-runs~1{workflowRunId}~1form-fields/get/responses : The operation 'listFormFieldValues' has more than one response with specified schema, only one schema will be used in the designer.```

Should we keep them or not then?